src/fab/_fab.scss

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.mdl-fab {
+  padding: 0;
+  width: 56px;
+  height: 56px;
+  border: none;
+  cursor: pointer;
+  overflow: hidden;
+  border-radius: 50%;
+  display: inline-flex;
+  justify-content: center;
+  background-color: $button-primary-color;
+  box-shadow: 0 1px 1.5px 0 rgba(0,0,0,0.12), 0 1px 1px 0 rgba(0,0,0,0.24);
+  -ms-user-select: none;
+  -moz-user-select: none;
+  -webkit-user-select: none;
+
+  &:active {
+    @include shadow-4dp();
+    background-color: $button-active-color;
+  }
+
+  &:focus:not(:active) {
+    @include focus-shadow();
+    background-color: $button-active-color;
+  }
+
+  &--mini {
+    height: 40px;
+    width: 40px;
+  }
+
+  &--colored {
```

Comment:
According to the spec, FABs are mainly accent coloured, so I don't think `--colored` should produce a primary-coloured FAB.

I think this is a good opportunity to reevaluate our colouring on buttons. What we have in v1 is:

- Button:
  - Default is grayscale.
  - `--colored` is primary.
  - `--accent` produces accent.
  - `--primary` produces primary.

- FAB:
  - Default is grayscale.
  - `--colored` is accent.
  - `--accent` produces accent.
  - `--primary` produces primary.

I propose the following instead for v2:

- Button:
  - Default is primary.
  - `--plain` or `--no-color` (whichever you prefer) is grayscale.
  - [Optionally, if we feel there's a sufficient use case for it] `--accent` produces accent.

- FAB:
  - Default is accent.
  - `--plain` or `--no-color` (whichever you prefer) is grayscale.
  - [Optionally, if we feel there's a sufficient use case for it] `--primary` produces primary.

WDYT?